src/Bindings/PluginManager.cpp

```diff
@@ -808,6 +808,25 @@ bool cPluginManager::CallHookLoginForge(cClientHandle & a_Client, AStringMap & a
 
 
 
+bool cPluginManager::CallHookMonsterMoved(cMonster & a_Monster, const Vector3d & a_OldPosition, const Vector3d & a_NewPosition)
+{
+	FIND_HOOK(HOOK_MONSTER_MOVED);
+	VERIFY_HOOK;
+
+	for (PluginList::iterator itr = Plugins->second.begin(); itr != Plugins->second.end(); ++itr)
+	{
+		if ((*itr)->OnMonsterMoved(a_Monster, a_OldPosition, a_NewPosition))
```

Comment:
I would suggest `for(auto * plugin : Plugins->second)`.
Or maybe merge https://github.com/cuberite/cuberite/pull/4027 first.

Reply:
Merged, @jonathathan please update this branch (`git fetch upstream master:master`, `git rebase master`, fix conflicts)